src/pkg/packager/prepare.go

```diff
@@ -249,6 +259,12 @@ func (p *Packager) FindImages() (imgMap map[string][]string, err error) {
 					// Otherwise, add to the list of images
 					message.Debugf("Imaged digest found: %s", descriptor.Digest)
 					validImages = append(validImages, image)
+					cosignArtifacts, err := utils.GetCosignArtifacts(image)
+					if err != nil {
+						message.WarnErrf(err, "Problem looking up cosign artifacts for %s: %s", image, err.Error())
+						erroredCosignLookups = append(erroredCosignLookups, image)
+					}
+					validImages = append(validImages, cosignArtifacts...)
```

Comment:
Instead of copying this code twice could we do it like the maybes as a separate section / loop to keep the logic self-contained?

Reply:
Moved to a separate loop, hopefully accomplishes this in the same way, also added a comment differentiating these as cosign artifacts.